pkg/apps/controller/deploymentconfig/deploymentconfig_controller.go

```diff
@@ -128,28 +128,44 @@ func (c *DeploymentConfigController) Handle(config *appsapi.DeploymentConfig) er
 			return err
 		}
 	}
+
+	// Never deploy with invalid or unresolved images
+	for i, container := range config.Spec.Template.Spec.Containers {
+		if strings.TrimSpace(container.Image) == "" {
```

Comment:
len() == 0

Reply:
I've noticed you like it :) I tend to write `== ""` as it say more what is meant here. compiler should optimize it away

Reply:
it is a consistency thing, I used to use `== ""` as well, but @smarterclayton always pointed it out in reviews so I learned to use len() ;-)

Reply:
harder to read, but as you wish ;)

Reply:
done